### DISCUSSION TITLE
The Japanese language OSPO Local Meetup quarterly report #2 (2023/04-06)

### DISCUSSION THREAD

**Discussion by kuwatm** [todogroup/ospology#337] [OSPO Stories]
In Japan, we hold a Japanese-language OSPO Local Meetup twice a month.
There are about 15 participants each time, but the 21st April was about half as many because of the timing was changed.
This quarter, all the meetings were online-only, but we plan to hold a real one in Shin-Yokohama on the 14th July.

<< Strategy & Hosting >> Leader Motai-san
(20230414 #5)
We discussed the relationship between business and Open Source. Again, we were reminded that the relationship is different depending on the business domain of the company. For example, SI, OSS consulting, hardware, etc. However, the challenges of increasing contributors to Open Source were common. And we could get consensus on that engineers/managers were less aware of their dependencies and have a difficulty convincing them why they should contribute.
(20230516 #7)
Participating members explained to each other how they engaged with Open Source for each business. As per Chatham House rules, I will skip the introduction here.
(20230609 #9)
The way forward is discussed. First step is to extract representative Open Source used by each company, and then analyze the relationship between representative Open Source and business, then analyze the relationship with contribution activities. Further discussions will be held at the next meeting in-person.

<< OSPO Launch & FAQ >> Leader Owada-san
(20230421 #6)
In the first round of FAQs, we first fielded Question candidates. For example, "What is OSPO?" , "Why is OSS management necessary?" , and "What do I start with to create an OSPO?". And we discussed whether the question was easy to understand for our readers, and which question to start with.
(20230526 #8)
For the question of "What are the benefits of using OSS?", we rose some answers and discussed the known benefits within OSPO, such as "the use of cutting-edge technology (software)", "talent acquisition opportunities", and "influence to standards". In that vein, we came to discuss how the benefits differ from each stages of OSPO.
(20230623 #10)
We were able to refine the text of the benefits by adding the previously extracted benefits of using Open Source to both "Maturity stages of OSPOs" and "The four core stages of open source strategy" from "A Deep Dive Into Open Source Program Offices" in the form of speech bubbles during our discussion.

The minutes or materials of all sessions are arranged below in Japanese and English for your perusal.
OpenChain-JWG/OSPO/OSPOLocalMeetup-Memo at master · OpenChain-Project/OpenChain-JWG ([github.com](http://github.com/))

**Comment by hiromotai7** [comment]
> << Strategy & Hosting >> Leader Motai-san

My leading section looks good to me.

Thank you!

**Comment by anajsana** [comment]
Thanks for the update OSPO Community Japan!
